### PR TITLE
Added hamburger menu

### PR DESCRIPTION
**Pull request by CrashtimeHD**
- maybe it looks a bit to clean and simple
- also the menu-background height isn't equal to the height of the actual site (used `min-height: 100vh` for now to fill atleast the screenheight when the menu wouldn't fill it) at scrolling the menu-background will end after the last menu-entry and show the actual site

### CODE REVIEW

**Comment by def-** [comment]
Doesn't look bad for an initial version. But the text is too big, fade thing can be removed, and has to look better in general.

File: www/_layouts/default.html [hunk 1]
```diff
@@ -34,8 +34,30 @@
...
+          <li><a href="/status/">Status</a></li>
+          <li><a href="/ranks/">Ranks</a></li>
+          <li><a href="/releases/">Releases</a></li>
+          <li><a href="//forum.ddnet.tw/">Forum</a></li>
+          <li><a href="/downloads/">Downloads</a></li>
+          <li><a href="/tournaments/">Tournaments</a></li>
+          <li><a href="/skins/">Skin Database</a></li>
+          <li><a href="/stats/">Statistics</a></li>
+          <li><a href="http://wiki.ddnet.tw/">Wiki</a></li>
+          {{ page.menu }}
```
Comment (def-) [inline www/_layouts/default.html]: You can just insert the div gap before this line instead of in every file.

**Comment by def-** [comment]
Is this ready to merge? Could you squash your changes into a single commit?

**Comment by CrashtimeHD** [comment]
So close this one and open a new pr with a single commit, or what should i do (I'm still a github noob)?

I have to fix a mistake in _post/... later before it can be merged.
The div gap is in every file so the gap after wiki is only there when a page submenu exist.
Otherwise there would be a gap everytime at the end of the menu when the page.menu is not defined/empty.

**Comment by def-** [comment]
I think there are some css tricks with before or after that you could use to make the gap without having to add a div.

**Comment by CrashtimeHD** [comment]
I will close this PR and open a new one since I'm not able to squash the commits
 #24

**Pull request closed without merging**